tests/gaslessTransaction.test.ts

```diff
@@ -0,0 +1,24 @@
+import {VerifiedWallet, Provider} from '../src'
+import VerifiedContract from '../src/contract/userop'
+import abi from '../src/abi/erc-20/erc20Abi.json'
+// Only for testing
+const apiKey ='a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799'
+const rpcUrl = "https://api.stackup.sh/v1/node/a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799"
+const paymasterUrl = 'https://api.stackup.sh/v1/paymaster/a6645c57b516720cc218e98a645887d59fa53b3e1e08fed0c58de17b3589b799'
+
+describe("userop gasless transaction", () => {
+
+    test('gasless transaction should succeed', async() => { 
+        const mnemonics = await VerifiedWallet.generateMnemonic()
+        const wallet = VerifiedWallet.importWallet(mnemonics)
+        const signer = wallet.setProvider(
+          Provider.stackUpProvider(rpcUrl)
+      )
+        const contract = new VerifiedContract(wallet.address, JSON.stringify(abi),signer)
+        const response =  await contract.callContract({ value:'0',apiKey,paymasterUrl})
```

Comment:
@herberthk 

I do not think you understood me again. The SDK contracts now are like the following code snippet. We are not passing bundler and paymaster urls to callContract at all now. You need to write a test case that initializes a SDK contract (eg, the Custody contract below) and calls a function on it (eg, the createVault function below on the Custody contract) and find another way of passing the userops config (eg, bundler and paymaster urls) to callContract

```
export default class Custody extends VerifiedContract {

    public contractAddress: string
    
    constructor(signer: VerifiedWallet, contractNetworkAddress: string) {
        
        const address = contractNetworkAddress        
        super(address, JSON.stringify(abi), signer)

        this.contractAddress = address
    }

    public async createVault(_creator: string, _id: string, options?: { gasPrice: number, gasLimit: number }): any {
        await this.validateInput(DATATYPES.STRING, _creator)
        await this.validateInput(DATATYPES.STRING, _id)
        return this.callContract(FUNCTIONS.CREATEVAULT, this.sanitiseInput(DATATYPES.BYTE32, _creator), _id, options)
    }
}
```

Reply:
@kallolborah I'm sorry, I didn't ask questions previously but let me do so now before I continue.
1. Do you want me to create a class that extends `userop` version of `VerifiedContract` that will be instantiated in tests?. Because as you know I can't use base `VerifiedContract` because of incompatibilities issues. Then pass userop config in `callContract` method internally in child class.
2. From example code `Custody extends VerifiedContract` Do you want modification to base `VerifiedContract` to support userop instead of creating userop version? I'm sorry but I need clarification on this Thanks in advance